QUICKSTART.md

```diff
@@ -0,0 +1,96 @@
+## StackHawk Setup
+
+1. If you do not have an existing account or application on StackHawk.
+   [Follow the steps getting set up](https://docs.stackhawk.com/hawkscan/#quickstart). Make sure to save your API key
+
+2. [Save the API Key](https://app.stackhawk.com/settings/apikeys)
+
+## GitHub Setup
+
+1. Create or use an existing GitHub [Repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository).
+
+2. Add the following secrets to your repository's [secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
+
+    - `API_KEY`: API Key from your stack hawk platform
```

Comment:
I think we just did a change to call this `HAWK_API_KEY` to be consistent in all the places

```suggestion
    - `HAWK_API_KEY`: API Key from your stack hawk platform
```